Automattic/jetpack
# IDC: "Safe Mode" admin bar button redesign

#### Changes proposed in this Pull Request:
* redesign the "Safe Mode" admin bar button

#### Jetpack product discussion
p9dueE-3pz-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

1. Rebuild the package if testing locally: `jetpack build packages/identity-crisis`. Install new version of the [Customizer plugin](https://gist.github.com/sergeymitr/f4257d64be6282e36b877a18a30d819f).
2. Connect Jetpack, go to Jetpack Debug and activate the IDC Simulator.
3. Go to "Jetpack Debug -> IDC Simulator", enable "IDC Simulation" and enter a "Spoof URL".
4. Edit a post and save to make the site sync. Wait a few seconds.
5. Open Jetpack Dashboard, you should see the IDC screen.
6. Click "Stay in Safe Mode". The page should reload, IDC screen should disappear, and "Jetpack Safe Mode" button should appear in the top right corner. Compare the button to the screenshot below.
7. Open a non-Jetpack page, the admin bar button should change label to "Sample Safe Mode", but look the same.

<img width="343" alt="Screen Shot 2022-01-25 at 3 28 43 PM" src="https://user-images.githubusercontent.com/1341249/151054436-93779859-192d-413b-a777-b5055babbc7b.png">
<img width="343" alt="Screen Shot 2022-01-25 at 5 20 57 PM" src="https://user-images.githubusercontent.com/1341249/151069475-c7e8b268-25b9-4dd3-a498-cdefd5300e78.png">
